lib/poller.py

```diff
@@ -120,9 +120,9 @@ def send_notifications(self, entries):
             if resp and resp.status_code == 200:
                 # Mark hold as processed:
                 self.redis_client.set_hold_processed(entry)
-            elif resp and resp.status_code == 400 and str(resp.content).find('Unable to find patron email') > -1:
-                # If the notification fails because the patron is missing an email,
-                # log the issue but do not retry
+            # If the notification fails because the patron is missing an email,
+            # log the issue but do not retry
+            elif resp.status_code == 400 and str(resp.content).find('Unable to find patron email') > -1:
```

Comment:
Is it possible `resp` is `None`?

Reply:
Good point! I will check that `resp is not None`